src/main/java/dev/morling/kccli/command/DescribeConnectorCommand.java

```diff
@@ -85,17 +93,37 @@ public void run() {
             config.add(new Tuple("  " + configEntry.getKey(), configEntry.getValue()));
         }
 
-        printTuples(Arrays.asList(new Tuple(ANSI_WHITE_BOLD + "Config" + ANSI_RESET, "")));
-        printTuples(config);
+        Tuple.print(Arrays.asList(new Tuple(ANSI_WHITE_BOLD + "Config" + ANSI_RESET, "")));
+        Tuple.print(config);
 
-        printTuples(Arrays.asList(new Tuple(ANSI_WHITE_BOLD + "Topics" + ANSI_RESET, "")));
+        Tuple.print(Arrays.asList(new Tuple(ANSI_WHITE_BOLD + "Topics" + ANSI_RESET, "")));
 
         List<Tuple> topics = new ArrayList<>();
 
         for (String topic : connectorTopics.entrySet().iterator().next().getValue().topics) {
             topics.add(new Tuple("", "  " + topic));
         }
-        printTuples(topics);
+        Tuple.print(topics);
```

Comment:
Let's put the tasks config above topics. That way the configs (connector + tasks) is next to each other which I feel is more consistent.